package.json

```diff
@@ -45,13 +45,13 @@
     "coveralls": "^3.0.2",
     "faucet": "0.0.1",
     "istanbul": "^1.1.0-alpha.1",
-    "jest": "^23.6.0",
+    "jest": "^24.1.0",
```

Comment:
please remove all carets in front of versions

Reply:
Removed carets from all versions.